source/containers.tex

```diff
@@ -9200,6 +9238,665 @@
 \end{codeblock}
 \end{itemdescr}
 
+\rSec2[inplace.vector]{Class template \tcode{inplace_vector}}
+
+\rSec3[inplace.vector.overview]{Overview}
+
+\pnum
+\indexlibraryglobal{inplace_vector}%
+An \tcode{inplace_vector} is a contiguous container.
+Its capacity is fixed and
+its elements are stored within the \tcode{inplace_vector} object itself.
+
+\pnum
+An \tcode{inplace_vector} meets all of the requirements
+of a container\iref{container.reqmts},
+of a reversible container\iref{container.rev.reqmts},
+of a contiguous container, and
+of a sequence container,
+including most of the optional sequence container requirements\iref{sequence.reqmts}.
+The exceptions are the
+\tcode{push_front},
+\tcode{prepend_range},
+\tcode{pop_front}, and
+\tcode{emplace_front}
+member functions, which are not provided.
+Descriptions are provided here only
+for operations on \tcode{inplace_vector} that
+are not described in one of these tables or
+for operations where there is additional semantic information.
+
+\pnum
+For any \tcode{N},
+\tcode{inplace_vector<T, N>::iterator} and
+\tcode{inplace_vector<T, N>::const_iterator}
+meet the constexpr iterator requirements.
+
+\pnum
+%FIXME: Should this instead say something like:
+%FIXME:     "For any \tcode{N} such that \tcode{N > 0} is \tcode{true}, ..."?
+%FIXME: Same for similar expressions added in P0843R14?
+For any \tcode{N > 0},
+if \tcode{is_trivial_v<T>} is \tcode{false}, then
+no \tcode{inplace_vector<T, N>} member functions
+are usable in constant expressions.
+
+\pnum
+Any member function of \tcode{inplace_vector<T, N>} that
```

Comment:
```suggestion
Any call to a member function of \tcode{inplace_vector} that
```

Drop `<T, N>` per above. Also, I think we're talking about the _behavior of a call_ to the function, not a property of the function itself.

Reply:
I think you're right that we're talking about the behavior of a call to the function, but maybe taking about "invoking" the function would be better? Please forward to LWG.

As for the `<T, N>`, leaving in for consistency.